### PR TITLE
Update nnop and pyop and add test

### PR DESCRIPTION
**Pull request by Chiiizzzy**

### CODE REVIEW

File: towhee/tests/mock_operators/__init__.py [hunk 1]
```diff
@@ -19,14 +19,16 @@
...
 
-ADD_OPERATOR_PATH = os.path.join(_MOCK_OPERATOR_DIR, "add_operator")
-SUB_OPERATOR_PATH = os.path.join(_MOCK_OPERATOR_DIR, "sub_operator")
-PYTORCH_CNN_OPERATOR_PATH = os.path.join(_MOCK_OPERATOR_DIR, "pytorch_cnn_operator")
-PYTORCH_IMAGE_CLASSIFICATION_OPERATOR_PATH = os.path.join(_MOCK_OPERATOR_DIR, "pytorch_image_classification_operator")
-PYTORCH_OBJECT_DETECTION_OPERATOR_PATH = os.path.join(_MOCK_OPERATOR_DIR, "pytorch_object_detection_operator")
-PYTORCH_TRANSFORMER_OPERATOR_PATH = os.path.join(_MOCK_OPERATOR_DIR, "pytorch_transformer_operator")
-PYTORCH_TRANSFORM_OPERATOR_PATH = os.path.join(_MOCK_OPERATOR_DIR, "pytorch_transform_operator")
-PYTORCH_TRANSFORMER_VEC_OPERATOR_PATH = os.path.join(_MOCK_OPERATOR_DIR, "pytorch_transformer_vec_operator")
+ADD_OPERATOR_PATH = os.path.join(_MOCK_OPERATOR_DIR, 'add_operator')
```
Comment (fzliu) [inline towhee/tests/mock_operators/__init__.py:22]: Do we really need all these global paths? If they are used only once (or twice), they should not be defined as file-level global variables.

File: towhee/operator/base.py [hunk 2]
```diff
@@ -87,10 +79,40 @@ def key(self, value):
...
+class PyOperator(Operator):
     """
-    PyTorch model operator base
+    Python function operator, no machine learning frameworks involved.
     """
+    def __init__(self):
+        super().__init__()
+        pass
 
     def train(self):
```
Comment (junjiejiangjjj) [inline towhee/operator/base.py:117]: Why py operator has train function?

File: towhee/tests/mock_operators/nn_operator/nn_operator.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    @property
+    def framework(self):
+        return self._framework
+
+    @framework.setter
+    def framework(self, framework: str):
+        self._framework = framework
+
+    def __call__(self):
+        print('I\'m an NNOperator.')
```
Comment (junjiejiangjjj) [inline towhee/tests/mock_operators/nn_operator/nn_operator.py:35]: Do  not use print function, we can use `engine_log`

**Pull request merged**